docs/src/modules/utils/helpers.js

```diff
@@ -77,6 +77,8 @@ function includePeerDependencies(deps, versions) {
   Object.assign(deps, {
     'react-dom': versions['react-dom'],
     react: versions.react,
+    '@emotion/core': versions['@emotion/core'],
+    '@emotion/styled': versions['@emotion/styled'],
```

Comment:
I would like to follow-up on https://github.com/mui-org/material-ui/pull/22435#issuecomment-696849469. 

@Andarist Does `@emotion/styled` have any singleton? Would it be safe if we make it a dependency of `@material-ui/style-engine`? We don't depend on the theme provider of emotion, we use our own.

Reply:
Styled pkg doesnt create any singletones - it only consumes them.

Reply:
Even if it doesn't have a singleton we should always be aware of potential duplicate bundling. Style engines shouldn't be direct dependencies if we want users to customize it. Otherwise they might use a different version than we do.

Reply:
> Otherwise they might use a different version than we do.

True, version can be different. Would it lead to issues?

Maybe we should be greedy like we were with @material-ui/style-engine (using a dependency over a peer dependency because simpler but makes using styled-components really harder)? We could start with the approach that can yield the simplest installation steps (@emotion/styled as a dependency) and based on the pain we hear from developers change the tradeoff.

Reply:
Another element, it seems that in the demos, we will do:

```jsx
import styled from '@material-ui/core/styled';
```

or something similar so we can benefit from auto-switch between em/sc, the theme provider of MUI, and the default theme's values.

Reply:
> True, version can be different. Would it lead to issues?

It would lead to multiple versions in the bundle: 

> Even if it doesn't have a singleton we should always be aware of potential duplicate bundling